src/test/java/com/gepardec/mega/rest/WorkerResourceTest.java

```diff
@@ -33,29 +34,42 @@
 @JwtSecurity(claims = {
         @Claim(key = "email", value = "test@gepardec.com")
 })
-class WorkerResourceTest {
+public class WorkerResourceTest {
 
     @InjectMock
-    MonthlyReportService monthlyReportService;
+    private MonthlyReportService monthlyReportService;
 
     @InjectMock
-    EmployeeService employeeService;
-
+    private EmployeeService employeeService;
 
     @InjectMock
-    UserContext userContext;
+    private UserContext userContext;
 
```

Comment:
Make all fields package-private, since IDE is confused about them not being instantiated explicitly